packages/cli/src/commands/compile.ts

```diff
@@ -118,7 +118,7 @@ const command: GluegunCommand = {
 
         switch (from_) {
           case 'mitosis':
-            json = parseJsx(data);
+            json = parseJsx(data, { typescript: !!path.match(/\.tsx?$/) });
```

Comment:
`generatorOpts` will hold the options for the generator you are currently using. You should be able to use its `generatorOpts.typescript` option and pass that through to `parseJsx`!

```suggestion
            json = parseJsx(data, { typescript: generatorOpts.typescript });
```